src/HttpClient/middlewares/cache.ts

```diff
@@ -198,7 +198,16 @@ export const cacheMiddleware = ({ type, storage }: CacheOptions) => {
         ? (data as Buffer).toString(responseEncoding)
         : data
 
-      const expiration = Date.now() + (maxAge - currentAge) * 1000
+      const now = Date.now()
+      const expiration = now + (maxAge - currentAge) * 1000
+
+      const alreadyExpired = expiration <= now
```

Comment:
I was confused at first here by this math, but it seems to me that given from L202:

`expiration - now = 1000 * (maxAge - currentAge)`

Then `expiration <= now` if, and only if, `maxAge <= currentAge`, which means `alreadyExpired` maybe could be simplified:


```suggestion
      const alreadyExpired = maxAge <= currentAge
```

Reply:
Oh, I see. I did not pay attention to what we had before :grimacing:

In that case I agree your change does not decrease the code clarity, since my eyes were drawn to that whole `const expiration = now + (maxAge - currentAge) * 1000` that was already there, and it doesn't seem like it will disappear haha. Let's move on.